src/screens/landingPage/components/advantages/styles.ts

```diff
@@ -126,3 +149,91 @@ export const HandsImage = styled(Box)`
     height: 100px !important;
   }
 `;
+
+export const TopLeftImage = styled(Box)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin-top: -228px;
+  background-image: url(${TopLeftImage1920});
+  width: 298px;
+  height: 388px;
+  @media (max-width: ${theme.breakpoints.values.md}px) {
+    background-image: url(${TopLeftImage905}) !important;
+    margin-top: -120px !important;
+    width: 158px !important;
+    height: 206px !important;
+  }
+  @media (max-width: ${theme.breakpoints.values.sm}px) {
+    background-image: url(${TopLeftImage360}) !important;
```

Comment:
**[Nesse componente](https://github.com/ajuda-ai-ufam/frontend/blob/development/src/screens/landingPage/components/turnIntoMonitor/styles.ts#L56)** tem um exemplo de como você pode colocar esses assets sem essas variações de tamanho com as seguintes props:
```css
  background-size: contain;
  background-repeat: no-repeat;
  background-position-y: center;
  background-position-x: initial;
  width: 600px !important;
  height: 500px !important;
```

Reply:
A prop `background-position-x`você pode setar com `initial` se for uma imagem que fica à esquerda e `right` se for uma da direita.